docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/client.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Client'
+description: 'Client is a HTTP client attaching tracing information to its requests.'
+weight: 02
+---
+
+# Client
+
+`Client` is an HTTP client constructor that attaches baggage header to its requests. Use it to include a context in HTTP requests so that  [Grafana pyroscope](https://grafana.com/oss/pyroscope/) can incorporate their results.
```

Comment:
Should baggage header be singular or plural? 
```suggestion
`Client` is an HTTP client constructor that attaches baggage header to its requests.
Use the client to include a context in HTTP requests so that [Grafana pyroscope](https://grafana.com/oss/pyroscope/) can incorporate their results.
```